docs/usage/strict.md

```diff
@@ -0,0 +1,91 @@
+
+
+You can use the `StrictStr`, `StrictBytes`, `StrictInt`, `StrictFloat`, and `StrictBool` types
```

Comment:
better to move these next to the types they go with, rather than being a section on their own.